### PR TITLE
added example usage of the katpoint target and antenna objects

### PR DESCRIPTION
**Pull request by fjoubert**

### CODE REVIEW

File: examples/get_target_descriptions.py [hunk 1]
```diff
@@ -45,6 +46,7 @@ def main():
     #         "track_duration":20.0
     #     },
     #     {
+    # input type:
```
Comment (ajoubertza) [inline examples/get_target_descriptions.py]: Where does this line come from?  It doesn't look like part of the example output.

Comment (fjoubert) [inline examples/get_target_descriptions.py]: Oops, that was a finger problem, I'll remove it.

**Comment by ludwigschwardt** [comment]
Sorry about that............. I always forget to submit the stupid review!!!

**Comment by fjoubert** [comment]
Thanks @ludwigschwardt!

**Pull request merged**